src/types/pathways-model.d.ts

```diff
@@ -8,6 +8,13 @@ declare module 'pathways-model' {
     states: {
       [key: string]: GuidanceState | BranchState | State;
     };
+    elm?: PathwayELM;
```

Comment:
I feel like this should remain optional.  It seems presumptive to assume that the app is only allowed to work when provided ELM, when we have the ability to convert CQL already.

Reply:
The idea is that once we have a pathway builder, it should have the ability to convert CQL to ELM directly, so at that point there won't be any need to create pathways manually and hand write CQL, and there shouldn't be any instances where the ELM isn't present.